### PR TITLE
 Styles: add CSS variables for use in demos and UI template gallery 

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

**Comment by EugeniyKiyashko** [comment]
Please, сheck the following places:
- https://github.com/nikkithelegendarypokemonster/devextreme-demos/blob/8836f2dd05324e566f0ed253a85eaff053223fab/MVCDemos/Views/Sortable/Customization.cshtml#L129
- https://github.com/nikkithelegendarypokemonster/devextreme-demos/blob/8836f2dd05324e566f0ed253a85eaff053223fab/MVCDemos/Views/FileUploader/CustomDropzone.cshtml#L3
- https://github.com/nikkithelegendarypokemonster/devextreme-demos/blob/8836f2dd05324e566f0ed253a85eaff053223fab/NetCoreDemos/Views/FileUploader/CustomDropzone.cshtml#L3
- https://github.com/nikkithelegendarypokemonster/devextreme-demos/blob/8836f2dd05324e566f0ed253a85eaff053223fab/NetCoreDemos/Views/Sortable/Customization.cshtml#L129
- https://github.com/nikkithelegendarypokemonster/devextreme-demos/blob/8836f2dd05324e566f0ed253a85eaff053223fab/NetCoreDemos/Views/TabPanel/Overview.cshtml#L26

We should change them as well.

**Comment by nikkithelegendarypokemonster** [comment]
ok sir will check these files

File: NetCoreDemos/wwwroot/css/DemosStyles/Tabs/Overview.css [hunk 1]
```diff
@@ -10,6 +10,10 @@
     max-width: 250px;
 }
 
+.tabpanel-demo .dx-tabpanel {
```
Comment (EugeniyKiyashko) [inline NetCoreDemos/wwwroot/css/DemosStyles/Tabs/Overview.css]: This style should be placed in other demo - TabPanel/Overview.

Comment (nikkithelegendarypokemonster) [reply NetCoreDemos/wwwroot/css/DemosStyles/Tabs/Overview.css]: sir this the correction for this : [Correction](https://github.com/DevExpress/devextreme-demos/pull/3163/commits/67981de22d858b8205dad4f7b245d6fba6b7ace4)

File: MVCDemos/Content/DemosStyles/Drawer/LeftOrRightPosition.css [hunk 2]
```diff
@@ -20,6 +21,7 @@
 .panel-list {
     height: 400px;
     padding-top: 12px;
+    background-color: var(--dx-color-main-bg);
```
Comment (EugeniyKiyashko) [inline MVCDemos/Content/DemosStyles/Drawer/LeftOrRightPosition.css:24]: Seem, you skipped change for '#content' selector in the line 51. Please, check it in both approaches .net, mvc and for both demos for Drawer component

Comment (EugeniyKiyashko) [reply MVCDemos/Content/DemosStyles/Drawer/LeftOrRightPosition.css:24]: #content elemnt should have background-color: var(--dx-component-color-bg);

Comment (nikkithelegendarypokemonster) [reply MVCDemos/Content/DemosStyles/Drawer/LeftOrRightPosition.css:24]: ok will check on this one

File: MVCDemos/Views/DropDownButton/Overview.cshtml [hunk 3]
```diff
@@ -148,10 +148,7 @@
 
     function applyColor($element, color) {
         if (color) {
-            $element.removeClass("dx-theme-text-color");
             $element.css("color", color);
-        } else {
-            $element.addClass("dx-theme-text-color");
```
Comment (EugeniyKiyashko) [inline MVCDemos/Views/DropDownButton/Overview.cshtml:154]: If color is not defined, we should remove inline style from $element, so you code should include else section with removing current color style property value.

File: MVCDemos/Views/FileUploader/CustomDropzone.cshtml [hunk 4]
```diff
@@ -74,12 +74,8 @@
 
     function toggleDropZoneActive(dropZone, isActive) {
         if(isActive) {
```
Comment (EugeniyKiyashko) [inline MVCDemos/Views/FileUploader/CustomDropzone.cshtml]: We can use .toggle method to add or remove 'dropzone-active' class:
```
dropZone.classList.toggle('dropzone-active', isActive);
```

File: NetCoreDemos/Views/FileUploader/CustomDropzone.cshtml [hunk 5]
```diff
@@ -74,12 +74,8 @@
 
     function toggleDropZoneActive(dropZone, isActive) {
         if(isActive) {
-            dropZone.classList.add("dx-theme-accent-as-border-color");
```
Comment (EugeniyKiyashko) [inline NetCoreDemos/Views/FileUploader/CustomDropzone.cshtml:77]: Use .toggle method here too.

File: MVCDemos/Content/DemosStyles/Drawer/LeftOrRightPosition.css (hunk 2 above)
Comment (nikkithelegendarypokemonster) [reply MVCDemos/Content/DemosStyles/Drawer/LeftOrRightPosition.css:24]: this is the correction on this one: [Correction](https://github.com/DevExpress/devextreme-demos/pull/3163/commits/a0ec1af541d02e97170949eada7dcbf55a27b514)

File: MVCDemos/Views/DropDownButton/Overview.cshtml (hunk 3 above)
Comment (nikkithelegendarypokemonster) [reply MVCDemos/Views/DropDownButton/Overview.cshtml:154]: this is the correction on this one: [Correction](https://github.com/DevExpress/devextreme-demos/pull/3163/commits/9e5cda76dc0f3d4c2790f1dfb878df4a97708710)

File: MVCDemos/Views/FileUploader/CustomDropzone.cshtml (hunk 4 above)
Comment (nikkithelegendarypokemonster) [reply MVCDemos/Views/FileUploader/CustomDropzone.cshtml]: this is the correction on this one: [Correction](https://github.com/DevExpress/devextreme-demos/pull/3163/commits/8b35563ece420243044d0428445a76dfa8d83aa5)

File: NetCoreDemos/Views/FileUploader/CustomDropzone.cshtml (hunk 5 above)
Comment (nikkithelegendarypokemonster) [reply NetCoreDemos/Views/FileUploader/CustomDropzone.cshtml:77]: the correction on this one: [Correction](https://github.com/DevExpress/devextreme-demos/pull/3163/commits/8b35563ece420243044d0428445a76dfa8d83aa5)

**Review by EugeniyKiyashko — DISMISSED** [review]

**Review by alexanderPolosatov — DISMISSED** [review]

**Review by alexanderPolosatov — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**